docs/topic_guides/splink_fundamentals/querying_splink_results.md

```diff
@@ -1,26 +1,42 @@
-# Retrieving and Querying Splink results with the SplinkDataFrame
+---
+tags:
+  - SQL
+  - Data Frames
+  - SplinkDataFrame
+toc_depth: 2
+---
+# Retrieving and Querying Splink Results
 
-Splink returns tables of results using a class called a `SplinkDataFrame`. e.g. when you run `df_predict = linker.predict()` `df_predict` is a `SplinkDataFrame`
+Once you have [created your linkage model](./settings.md), [trained your parameters](../demos/tutorials/04_Estimating_model_parameters.html) and [predicted your match probabilities](../demos/tutorials/05_Predicting_results.html) you may want to run queries on your results. Splink provides several methods for interacting with the tables produced in this process.
```

Comment:
demos links should be e.g.
`../../demos/tutorials/04_Estimating_model_parameters.html`